godtools/App/Share/Views/TransparentModal/TransparentModalView.swift

```diff
@@ -68,6 +75,16 @@ class TransparentModalView: UIViewController {
         // modalView
         addModalView(modalView: modalView)
     }
+    
+    private func setupBinding() {
+        
+        overlayButton.addTarget(self, action: #selector(handleBackgroundTapped), for: .touchUpInside)
```

Comment:
Move into the viewDidLoad and keep the setupBinding() for any viewModel binding.